src/test/resources/processor_test_files/2116_ArrayHashCodeAndToString/ArrayHashCodeAndToString.java.expected

```diff
@@ -1,12 +1,14 @@
 
 // Test for rule s2116
 
+import java.util.Arrays;
+
 public class ArrayHashCodeAndToString {
 
 	// Tests from https://rules.sonarsource.com/java/type/Bug/RSPEC-2116
 	public static void main( String[] args ) {
-		String argStr = java.util.Arrays.toString(args); // Noncompliant
-		int argHash = java.util.Arrays.hashCode(args); // Noncompliant
+		String argStr = Arrays.toString(args); // Noncompliant
+		int argHash = Arrays.hashCode(args); // Noncompliant
```

Comment:
Example of how this update changes the output of a processor (note also that `java.util.Arrays` is imported at the top of the file)